recipes-tag/frozendict/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "frozendict" %}
+{% set version = "1.2" %}
+{% set sha256 = "774179f22db2ef8a106e9c38d4d1f8503864603db08de2e33be5b778230f6e45" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 2
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - frozendict
+
+about:
+  home: https://github.com/slezica/python-frozendict
```

Comment:
This package seems to be an implementation of [this SO answer](https://stackoverflow.com/a/28452633), but with a few additions. Good to have it as a standalone library anyway.